.gitignore

```diff
@@ -20,3 +20,6 @@ omni_exchange-1.1.0.gem
 omni_exchange-1.1.1.gem
 omni_exchange-1.2.0.gem
 omni_exchange-1.3.0.gem
+omni_exchange-1.4.0.gem
+omni_exchange-1.5.0.gem
+omni_exchange-1.6.0.gem
```

Comment:
If gitignore supports wild cards this might be a good place to use them :D

Reply:
@Metallion I didn't even know that was a thing! I'll look into it... thank you for the suggestion!

Reply:
I think `*.gem` is fine.
By the way, should we ignore this file? It is not generated in my environment.

Reply:
I think `*.gem` is fine.
By the way, should we ignore this file? It is not generated in my environment.